docs/data/charts/overview/overview.md

```diff
@@ -16,21 +16,21 @@ The `@mui/x-charts` is an MIT library for rendering charts relying on [D3.js](ht
 And, like other MUI X components, charts are production-ready components that integrate smoothly into your app.
 
 With a high level of customization, MUI X Charts provides on three levels of customization layers: **single components** with great defaults, extensive **configuration props**, and **subcomponents** for flexible composition.
-Additionally, you can also use all the MUI System tools, such as the theme override or the `sx` prop.
+Additionally, you can also use all the [MUI System](https://mui.com/system/getting-started/) tools, such as the theme override or the `sx` prop.
```

Comment:
It wasn't super clear to me what "use all the MUI System tools" means, I imagine it won't for developers as well.

---

**Off-topic.** Today, it's kind of crap for the charts to [have](https://unpkg.com/browse/@mui/x-charts@7.0.0-alpha.5/package.json) a dependency on `@mui/system` vs. say Tailwind CSS or even better, inline style for someone that is outside of the MUI ecosystem and is looking for great charts. Now, we are still at a phase where charts have enough room to grow into the MUI ecosystem, so it's not important and even helping (it's important for the data grid though).

I believe there are two main changes we can/should make that will make this great:

- @mui/system moving to runtime-less CSS-in-JS. I would expect https://bundlephobia.com/package/@mui/system@5.15.2 to go from 20KB gzipped to 2KB gzipped if we play this well. I'm not exactly sure how to keep the createTheme helpers super small, but there must be a way. For developers, assuming they configure it correctly (as they would have to configure Tailwind CSS). It means that there is almost no downside for folks who don't want to use MUI System. For example, when using recharts or highcharts, everything is inline style. Now, with a canvas implementation of charts, we would have to get there, making @mui/system an optional peer dependency (peer because I imagine it will host a context, so a singleton). For example, [react-data-grid is on Linaria](https://github.com/adazzle/react-data-grid/blob/main/src/cellRenderers/renderCheckbox.tsx#L1).
- @mui/material moving to not longer having a default theme, forcing the developer to configure `@mui/system` and consume directly from it. For the charts, data grid, this would mean that importing from a Material UI component wouldn't come with as much overhead as it does today. e.g. solving https://github.com/adazzle/react-data-grid/issues/1999

I'm adding @brijeshb42 and @mnajdova for context so they are aware of the need for MUI X, and what's at stake.

Reply:
Thanks for the mention, it's good to know. I haven't seen the implementation, so I'll just ask here, do the charts use any Material UI component or it's only about the MUI System? What you are proposing makes sense long term 👍